build.html

```diff
@@ -145,9 +145,9 @@ <h1>
           </div>
         </div>
         <div id="request-metadata">
-          <h2>
+          <h1>
```

Comment:
Will this pass the accessibility scan?

Reply:
Might want to deploy your branch and then run this job: https://bcda-ci.adhocteam.us/job/BCDA%20-%20Static%20Site%20508%20Compliance/
Up to you; even if it does fail, we can fix it later.

Reply:
This should pass the scan as it doesnt seem to have any issues with the headers. But I will deploy to dev and run the quality gates to see the results.